provided-template.html

```diff
@@ -0,0 +1,453 @@
+  <title>Palo Alto United School District Newsletter</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <link href="https://opensource.keycdn.com/fontawesome/4.7.0/font-awesome.min.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Fira+Sans" rel="stylesheet">
+</head>
+  <body style="margin: 0; padding: 0;background-color:#EEEEEE;font-family: 'Fira Sans', sans-serif;">
+    <!-- Styles used through the page-->
+    <style>
+
+    @import url('https://fonts.googleapis.com/css?family=Fira+Sans');
+
+      h2 {
+        color: #387C2C;
+        font-weight:500;
+        font-size: 27px;
+        font-family: 'Fira Sans', sans-serif;
+        margin:0px;
+        line-height: 32px;
+      }
+      h3 {
+        color:#444444;
+        text-transform: uppercase;
+        font-weight: 600;
+        font-size: 18px;
+        font-family: 'Fira Sans', sans-serif;
+      }
+      h4 {
+        font-size: 15px;
+        font-family: 'Fira Sans', sans-serif;
+        color: white;
+        font-weight: 500;
+        line-height: 10px;
+      }
+      .align-left {
+        text-align:left;
+      }
+      p {
+        color: #606160;
+        font-weight: 300;
+        font-family: 'Fira Sans', sans-serif;
+        font-size:14px;
+        line-height: 26px;
+        margin:0px;
+      }
+      .read-more a {
+        text-decoration: none;
+        font-size:13px;
+        font-weight: 600;
+        color: #347928;
+        font-family: 'Fira Sans', sans-serif;
+      }
+      .border-top {
+        border-top: 1px solid #CBCCCB;
+      }
+      .tags {
+        font-style: italic;
+        font-family: 'Fira Sans', sans-serif;
+      }
+      .position {
+        color: white;
+        font-size: 14px;
+        font-weight: 300;
+        font-style: normal;
+      }
+      .name {
+        margin: 12px 0px 0px 0px;
+        font-weight: 400;
+        font-style: normal;
+      }
+      .events p {
+        line-height: 20px;
+      }
+      .fa {
+        font-size: 30px;
+        text-align: right;
+        padding-top:20px;
+      }
+      .social-icons {
+        color: #444444;
+        height: 36px;
+        width: 36px;
+        margin: 0em;
+        padding: 0em;
+        display: inline-block;
+        vertical-align: text-top;
+        border: 1px solid #444444;
+        text-align: center;
+        border-radius: 50%;
+        line-height: 36px;
+        font-size: 21px;
+        margin-right:25px;
+        font-weight: 600;
+        padding-top:0px;
+      }
+      .social-icons:hover {
+        color: #387C2C;
+      }
+      .social-bar {
+        text-align: center;
+      }
+      .social-bar-links {
+        text-decoration: none;
+        color: #858585;
+        margin-top:20px;
+      }
+      .social-bar-links:hover {
+        color: #387C2C;
+      }
+
+    </style>
+
+    <table align="center" border="0" cellpadding="0" cellspacing="0" width="960" bgcolor="white">
```

Comment:
I didn't catch this at first, but this width is to wide for email. The max possible width allowed for most clients is 650, but a common best practice/good standard is 600px.

This only applies to email.